spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaDiscoveryClientConfiguration.java

```diff
@@ -40,19 +42,27 @@
  * @author Jon Schneider
  * @author Jakub Narloch
  * @author Olga Maciaszek-Sharma
+ * @author Tim Ysewyn
  */
 @Configuration
 @EnableConfigurationProperties
 @ConditionalOnClass(EurekaClientConfig.class)
 @ConditionalOnProperty(value = "eureka.client.enabled", matchIfMissing = true)
 @ConditionalOnDiscoveryEnabled
+@ConditionalOnBlockingDiscoveryEnabled
 public class EurekaDiscoveryClientConfiguration {
 
 	@Bean
 	public Marker eurekaDiscoverClientMarker() {
```

Comment:
is this still going to be used anywhere? If not, maybe we should deprecate it?

Reply:
Also I have seen a discussion regarding the marker use is going on [here](https://github.com/spring-cloud/spring-cloud-zookeeper/pull/235/files#r324941423) - whatever the conclusion of this discussion is, please make sure to apply it here as well.